packages/manager/src/features/Databases/DatabaseCreate/DatabaseCreate.tsx

```diff
@@ -546,14 +547,16 @@ const DatabaseCreate: React.FC<{}> = () => {
             </RadioGroup>
           </FormControl>
           <Grid item xs={12} md={8}>
-            <Notice success className={classes.notice}>
-              <strong>
-                Notice: There is no charge for database clusters during beta.
-              </strong>{' '}
-              You will be notified before the beta period ends and database
-              clusters are subject to charges.{' '}
-              <Link to="https://www.linode.com/pricing/">View pricing</Link>.
-            </Notice>
+            {flags.databaseBeta ? (
+              <Notice informational className={classes.notice}>
+                <strong>
+                  Notice: There is no charge for database clusters during beta.
+                </strong>{' '}
+                Database clusters will be subject to charges when the beta
+                period ends on May 2nd, 2022.{' '}
+                <Link to="https://www.linode.com/pricing/">View pricing</Link>.
```

Comment:
Here's a link directly to the database pricing

```suggestion
                <Link to="https://www.linode.com/pricing/#databases">View pricing</Link>.
```